### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#517] [Galaxy Trivia] [answered]
💂🌧️
🐋🐋🐋
😠⛽🚗

**Comment by peterbaccaro** [comment]
1. ENGLAND
2. WALES
3. MADAGASCAR

**Reply by ParanoidUser** [reply]
Nearly there! The guard emoji in the first country might be a bit misleading. What if we add 🐋🐋🐋 (and 2 others) to it?

**Reply by peterbaccaro** [reply]
Then it would be GREAT BRITAIN.

**Reply by ParanoidUser** [reply]
Great! **your first edit was right**. Know let's get short form and add 🌧️ to it. What is the result? 😉

**Reply by peterbaccaro** [reply]
Ah, **UKRAINE**. 😆

**Reply by ParanoidUser** [reply]
High five! 😉

**Comment by thuongtruong1009** [comment, marked as answer]
The answer to the riddle is "England".

The clues are as follows:

💂🌧️ - The first emoji refers to the iconic symbol of England, a British guard, and the second emoji likely represents the rainy weather that is common in England.

🐋🐋🐋 - These three whale emojis can be interpreted as a reference to the River Thames, which flows through England.

😠⛽🚗 - These emojis can be interpreted as a representation of the frustration caused by traffic congestion in England, especially in cities like London. The angry face likely refers to the frustration, the gas pump represents the high cost of petrol in England, and the car represents the traffic.

**Comment by ErKiran** [comment]
1. United Kingdom (UK) - represented by the emoji of a British Guard (💂) and rainy weather (🌧️), which is often associated with the UK.
2. Iceland - represented by three whale emojis (🐋🐋🐋), a country is known for its whale-watching opportunities.
3. Saudi Arabia - represented by an angry face emoji (😠), a fuel pump emoji (⛽), and a car emoji (🚗), which could mean the country's association with oil production and transportation.
